### ISSUE TITLE
UE5.0 compatibility 

### ISSUE THREAD

**Issue by neoteknic** [CodeSpartan/UE4TcpSocketPlugin#16]
Is it possible to make it compatible with UE5.0 by changing EngineVersion:"5.0.0" ? Or will it require a lot of work ?

**Comment by CodeSpartan** [comment]
Could work. Why don't you give it a try?
You can delete the line with the engine version entirely when trying.

**Comment by lordrex34** [comment]
The plugin works with UE5.

**Comment by rforman9** [comment]
@lordrex34, can you elaborate on your above comment "The plugin works with UE5." 
I'm getting this message when I try to enable the plugin in UE5:
```
---------------------------
Missing Modules
---------------------------
The following modules are missing or built with a different engine version:

  TcpSocketPlugin

Engine modules cannot be compiled at runtime. Please build through your IDE.
---------------------------
OK   
---------------------------
```
Can you explain how you got it to build/compile?

**Comment by lordrex34** [comment]
@rforman9 change the version in the plugin and build it with UE5 EA.
Additionally if it doesn't work (it worked for me), you may try to investigate with Rider.

**Comment by rforman9** [comment]
The good news is I got it to compile.  The bad news is that I'm getting an error at runtime when it's called from this [Face Capture with Android](https://forums.unrealengine.com/t/face-capture-with-android-metahuman-download-links-free-open-source-demo/234927) App.
`Couldn't connect to server. TcpSocketConnection.cpp: line 410`
While trying to debug it in VS2019, It looks like it's trying to connect using a default ip address of 192.168.1.27,  and not using the ip address that I set in the unreal app.  Still trying to figure out exactly where the disconnect is though.  This app and plugin run without errors in UE4.26.
